### PR TITLE
sample test 1 [pr]

### PR DESCRIPTION
**Pull request by pvu-github**
this is just a test

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
:white_check_mark: A review job has been created and sent to the PullRequest network.  
*****  
Check the status or cancel PullRequest code review [here](https://peter-2813b.pullrequest.tech/dash/projects/1/pvu-github/TypeScript-React-Starter/reviews) - or - cancel by adding **[!pr]** to the title of the pull request.

**Comment by pvu-github** [comment]
Hey @PullRequest, what do you think about these changes?

**Comment by pvu-github** [comment]
Hey @pullrequest, what do you think about these changes?

**Comment by pvu-github** [comment]
Hey @pullrequest, what do you think about these changes?

File: src/App.tsx [hunk 1]
```diff
@@ -12,9 +12,9 @@ function App() {
         <h2>Welcome to React</h2>
       </div>
```
[the remark this answers is not in the document]
Comment (pvu-github) [reply src/App.tsx:15]: What do you think about these changes?

**Comment by pvu-github** [comment]
Hey Peter R, what do you think about this change?

File: src/App.tsx [hunk 2]
```diff
@@ -12,9 +14,9 @@ function App() {
...
       <p className="App-intro">
-        To get started, edit <code>src/App.tsx</code> and save to reload.
+        To get started, we want to edit <code>src/App.tsx</code> and save to reload.
```
[the remark this answers is not in the document]
Comment (pvu-github) [reply src/App.tsx:17]: replying to test comment

**Comment by pvu-github** [comment]
This is simulation a bot response. Check this out PullRequest

File: src/App.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pvu-github) [reply src/App.tsx:17]: replying to a test comment as a BOT

File: src/App.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (pvu-github) [reply src/App.tsx:15]: replying as a BOT - testing

**Pull request closed without merging**